rpc/server.go

```diff
@@ -79,9 +79,9 @@ func (s *Server) Start() <-chan error {
 		methodsOk := handlers.AllowedMethods([]string{"GET", "HEAD", "POST", "PUT", "OPTIONS"})
 		originsOk := handlers.AllowedOrigins([]string{"*"})
 
-		log.Infof("starting RPC server on http://localhost:%d", s.port)
+		log.Infof("starting RPC server on http://127.0.0.1:%d", s.port)
 
-		if err := http.ListenAndServe(fmt.Sprintf(":%d", s.port), handlers.CORS(headersOk, methodsOk, originsOk)(r)); err != nil { //nolint:lll
```

Comment:
The logged message was saying the port is only accessible from  `localhost`, while the actual `ListenAndServe` call was using all interfaces. I changed the code to match the logged message.